mne_connectivity/envelope.py

```diff
@@ -145,8 +155,33 @@ def envelope_correlation(data, combine='mean', orthogonalize="pairwise",
             if absolute:
                 corr = np.abs(corr)
             corr = (corr.T + corr) / 2.
+
         corrs.append(corr)
         del corr
 
+    # ravel from 2D connectivity into 1D array
+    corrs = [corr.flatten() for corr in corrs]
+
+    n_epochs = len(corrs)
     corr = fun(corrs)
-    return corr
+
+    # create the connectivity container
+    times = None
+
+    # create epochs axis
+    if combine is not None:
+        corr = corr[np.newaxis, ...]
```

Comment:
From an API standpoint, I wonder if we should get rid of `combine` as a function argument here and add it as a method of `EpochsTemporalConnectivity` that you can call to combine to obtain a TemporalConnectivity.

And it actually seems like there are some classes missing, like EpochConnectivity and then Connectivity that have no time space or frequency dims.

In other words, I think this should return an instance of EpochConnectivity, and that class should have a method `.combine` that returns a `Connectivity`

Reply:
Getting rid of `combine` makes sense to me, and I think I can do that on the next PR to clean this one up if you don't mind?

Then I have preliminarily added the EpochConnectivity and Connectivity data structures in now.

Reply:
Yes I can live with it in a subsequent PR